app/src/NativeBase/Components/Checkbox.tsx

```diff
@@ -0,0 +1,44 @@
+/**
+ * @file CheckBox component for the STA Volunteer App
+ */
+
+import { Pressable } from 'native-base'
+import React from 'react'
+import { Ionicons } from '@expo/vector-icons'
+
+type CheckboxProps = {
+  onChange: (value: string[]) => void
+  checked: boolean
+}
+
+/**
+ * Component that displays a checkbox that can be checked or unchecked.
+ *
+ * @param { CheckboxProps} props The component props
+ * @param {value: string[] => void} props.onChange function called when the checkbox is pressed
+ * @param {boolean} props.checked whether the checkbox is checked or not
+ * @returns {React.ReactElement} Component
+ */
+
+const Checkbox: React.FC<CheckboxProps> = ({ onChange, checked }) => {
+  return (
+    <Pressable
+      onPress={() => onChange([])}
+      flexDirection={'row'}
```

Comment:
Tiny thing, but from looking at examples in NativeBase docs, I think normal style for setting text properties here and below is just to use double quotes e.g. `flexDirection="row"` (see e.g. examples at https://docs.nativebase.io/pressable)

Reply:
I have updated the Pressable properties to use double-quotes instead.